util/MsSql/backup-db.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/sh
+BACKUP_INTERVAL=${BACKUP_INTERVAL:-next day}
+BACKUP_INTERVAL_FORMAT=${BACKUP_INTERVAL_FORMAT:-%Y-%m-%d 00:00:00}
 
 while true
 do
   # Sleep until next day
-  [ "$1" = "loop" ] && sleep $((24 * 3600 - (`date +%_H` * 3600 + `date +%_M` * 60 + `date +%_S`)))
+  if [ "$1" = "loop" ]; then
+    interval_start=`date -u "+${BACKUP_INTERVAL_FORMAT}" -d "${BACKUP_INTERVAL}"`
```

Comment:
Remove `-u` option so that calculation is made according to the container's timezone ?
As with can enforce it with the `TZ` environment variable, we should consider it, and not overwrite it.

Reply:
When removing `-u` for setting this, the sleep calculation was calculating negative numbers.

I did find a way around that that honors the `TZ` env var if set

Reply:
Strange, by curiosity, I gave a try without `TZ` set, and with `TZ` set to positive and negative timezones, I did not manage to get negative numbers, and do not really see how we could.
But of course we can keep the additional `%z` you added, it does not hurt.